Managers/FrameMan.cpp

```diff
@@ -1,2651 +1,1391 @@
+		if (m_StoreNetworkBackBuffer) {
+			// Blit all four internal player screens onto the backbuffer
+			for (short i = 0; i < c_MaxScreenCount; i++) {
+				int dx = 0;
+				int dy = 0;
+				int dw = m_pBackBuffer8->w / 2;
+				int dh = m_pBackBuffer8->h / 2;
+
+				switch (i) {
+					case 1:
+						dx = dw;
+						break;
+					case 2:
+						dy = dh;
+						break;
+					case 3:
+						dx = dw;
+						dy = dh;
+						break;
+					default:
+						break;
+				}
+				//m_TargetPos[i] = g_SceneMan.GetOffset(i);
+
+				m_NetworkBitmapIsLocked[i] = true;
+				blit(m_pNetworkBackBufferIntermediate8[m_NetworkFrameCurrent][i], m_pNetworkBackBufferFinal8[m_NetworkFrameCurrent][i], 0, 0, 0, 0, m_pNetworkBackBufferFinal8[m_NetworkFrameCurrent][i]->w, m_pNetworkBackBufferFinal8[m_NetworkFrameCurrent][i]->h);
+				blit(m_pNetworkBackBufferIntermediateGUI8[m_NetworkFrameCurrent][i], m_pNetworkBackBufferFinalGUI8[m_NetworkFrameCurrent][i], 0, 0, 0, 0, m_pNetworkBackBufferFinalGUI8[m_NetworkFrameCurrent][i]->w, m_pNetworkBackBufferFinalGUI8[m_NetworkFrameCurrent][i]->h);
+				m_NetworkBitmapIsLocked[i] = false;
+
+				// Draw all player's screen into one
+				if (g_UInputMan.KeyHeld(KEY_5)) {
+					stretch_blit(m_pNetworkBackBufferFinal8[m_NetworkFrameCurrent][i], m_pBackBuffer8, 0, 0, m_pNetworkBackBufferFinal8[m_NetworkFrameReady][i]->w, m_pNetworkBackBufferFinal8[m_NetworkFrameReady][i]->h, dx, dy, dw, dh);
+				}
+			}
+
+#if defined DEBUG_BUILD || defined MIN_DEBUG_BUILD
+			if (g_UInputMan.KeyHeld(KEY_1)) {
+				stretch_blit(m_pNetworkBackBufferFinal8[0][0], m_pBackBuffer8, 0, 0, m_pNetworkBackBufferFinal8[m_NetworkFrameReady][0]->w, m_pNetworkBackBufferFinal8[m_NetworkFrameReady][0]->h, 0, 0, m_pBackBuffer8->w, m_pBackBuffer8->h);
+			}
+			if (g_UInputMan.KeyHeld(KEY_2)) {
+				stretch_blit(m_pNetworkBackBufferFinal8[1][0], m_pBackBuffer8, 0, 0, m_pNetworkBackBufferFinal8[m_NetworkFrameReady][1]->w, m_pNetworkBackBufferFinal8[m_NetworkFrameReady][1]->h, 0, 0, m_pBackBuffer8->w, m_pBackBuffer8->h);
+			}
+			if (g_UInputMan.KeyHeld(KEY_3)) {
+				stretch_blit(m_pNetworkBackBufferFinal8[m_NetworkFrameReady][2], m_pBackBuffer8, 0, 0, m_pNetworkBackBufferFinal8[m_NetworkFrameReady][2]->w, m_pNetworkBackBufferFinal8[m_NetworkFrameReady][2]->h, 0, 0, m_pBackBuffer8->w, m_pBackBuffer8->h);
+			}
+			if (g_UInputMan.KeyHeld(KEY_4)) {
+				stretch_blit(m_pNetworkBackBufferFinal8[m_NetworkFrameReady][3], m_pBackBuffer8, 0, 0, m_pNetworkBackBufferFinal8[m_NetworkFrameReady][3]->w, m_pNetworkBackBufferFinal8[m_NetworkFrameReady][3]->h, 0, 0, m_pBackBuffer8->w, m_pBackBuffer8->h);
+			}
+#endif
+
+			// Rendering complete, we can finally mark current frame as ready
+			// This is needed to make rendering look totally atomic for the server pulling data in separate threads
+			m_NetworkFrameReady = m_NetworkFrameCurrent;
+			m_NetworkFrameCurrent = (m_NetworkFrameCurrent == 0) ? 1 : 0;
+
+			/*for (int i = 0; i < c_MaxScreenCount; i++)
```

Comment:
Kill this commented code. The `m_NetworkBitmapIsLocked` handling is done above on line 578.